src/geo/leaflet/leaflet-tiled-layer-view.js

```diff
@@ -37,7 +25,7 @@ LeafletTiledLayerView.prototype = _.extend(
   LeafletLayerView.prototype,
   {
     _createLeafletLayer: function () {
-      return new L.TileLayer(this.model.get('urlTemplate'), generateLeafletLayerOptions(this.model));
+      return new L.TileLayer(_getUrlTemplate(this._isHdpi(), this.model), _generateLeafletLayerOptions.call(this, this.model));
```

Comment:
Are u sure the `.call(this,` is needed?

Reply:
`_generateLeafletLayerOptions` is a pure function, it receives a layerModel and returns a options object, it wont use the js context at all right?